.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "src/home_robot/home_robot/perception/detection/grounded_sam/Grounded-Segment-Anything"]
 	path = src/home_robot/home_robot/perception/detection/grounded_sam/Grounded-Segment-Anything
 	url = https://github.com/IDEA-Research/Grounded-Segment-Anything
+[submodule "src/home_robot/home_robot/perception/minigpt4/MiniGPT-4"]
```

Comment:
Do we want to add this as a submodule?

Reply:
Oof I have no idea where this came from -- I'll remove it from the commit history

Reply:
thanks. it is in the demo grasp branch but i'm not sure we want to keep it